### PR TITLE
Refactor Agents CLI/Config

### PR DESCRIPTION
**Pull request by CryptoMaxPlanck**
**Description**
The command line for the agents has many fields. This PR reduces the amount of command line args and moves them into the config, as well as streamlines a lot of configuration elements.

### CODE REVIEW

**Comment by CryptoMaxPlanck** [comment]
Need to merge #1051, test with a new helmfile deploy in the helmfile repo, then to add the new configs into the helm charts in this repo.

**Review by joecroninallen — APPROVED** [review]

File: agents/agents/agentsintegration/agentsintegration_test.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import (
 	"github.com/brianvoe/gofakeit/v6"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/synapsecns/sanguine/agents/agents/executor"
-	executorCfg "github.com/synapsecns/sanguine/agents/agents/executor/config"
 	"github.com/synapsecns/sanguine/agents/agents/guard"
 	"github.com/synapsecns/sanguine/agents/agents/notary"
+	executor2 "github.com/synapsecns/sanguine/agents/config/executor"
```
Comment (trajan0x) [inline agents/agents/agentsintegration/agentsintegration_test.go:10]: can we get rid of these "2" suffixes, if it conflicts w/ a var var should prob be renamed. If it's another package (the case here, just do executorConfig for clarity)

File: agents/config/executor/db.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+type DBConfig struct {
+	// Type is the type of database. This can be either "sqlite" or "mysql".
+	Type string `yaml:"type"`
+	// Source is the source of the database. This can be either a path to a sqlite database or a mysql database url.
+	Source string `yaml:"source"`
+}
+
+// IsValid asserts the database connection is valid.
+func (d *DBConfig) IsValid(_ context.Context) (ok bool, err error) {
+	if d.Type != "sqlite" && d.Type != "mysql" {
```
Comment (trajan0x) [inline agents/config/executor/db.go:18]: use dbcommon consts here

**Review by trajan0x — CHANGES_REQUESTED** [review]

File: agents/agents/agentsintegration/agentsintegration_test.go (hunk 1 above)
Comment (CryptoMaxPlanck) [reply agents/agents/agentsintegration/agentsintegration_test.go]: done

File: agents/config/executor/db.go (hunk 2 above)
Comment (CryptoMaxPlanck) [reply agents/config/executor/db.go]: done

**Review by trajan0x — APPROVED** [review]

**Pull request merged**